examples_tests/35.TAA/quad.vert

```diff
@@ -0,0 +1,12 @@
+#version 430 core
```

Comment:
A. We have a full screen triangle extension to use
B. We shouldn't be even using that anymore because it should be done in compute shaders

Reply:
A. it uses full screen triangle
B. yea, i'll rewrite it as CS when it goes as irrbaw extension